src/DotNetCore.CAP.MongoDB/IStorage.MongoDB.cs

```diff
@@ -59,7 +59,47 @@ public async Task InitializeAsync(CancellationToken cancellationToken)
                     cancellationToken: cancellationToken);
             }
 
-            _logger.LogDebug("Ensuring all create database tables script are applied.");
+            var received_message_index_names = new string[] {

```

Comment:
Please using **Lower Camel Case** to replace **Under Score Case** for naming convention. 
`var receivedMessageIndexNames=`